pkg/otlp/config_serverless_noop.go

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+// +build serverless
+
+package otlp
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// IsEnabled no-op
```

Comment:
I would just comment it with something like:

// IsEnabled always returns false since OTLP is disabled in the serverless agent flavor.